### ISSUE TITLE
Loss Function?

### ISSUE THREAD

**Issue by haoshuai714** [mwray/Semantic-Video-Retrieval#2]
Hi Michael:  What loss function did you use in this paper?I can't find the related code of loss function in the code. Could you please provide the relevant codes?

**Comment by mwray** [comment]
Thanks for the interest in the code!

The loss function that we used for training the models for semantic similarity are the same as those used originally. Please see the CE codebase [here](https://github.com/albanie/collaborative-experts) and the JPoSE codebase [here](https://github.com/mwray/Joint-Part-of-Speech-Embeddings) for more details. We change the sampling method during training so that instead of using the corresponding item from the opposite modality as a positive, we instead sample from all of the items which have a similarity over a specified threshold.

For more info, please check section 4.3 in [the paper](https://mwray.github.io/SSVR/SSVR.pdf) for more details.

Thanks,

Michael
